core/src/main/java/io/questdb/griffin/engine/ops/AlterOperationBuilder.java

```diff
@@ -105,7 +107,7 @@ public void ofAddColumn(
             int indexValueBlockCapacity
     ) {
         assert columnName != null && columnName.length() > 0;
-        extraStrInfo.add(columnName);
+        extraStrInfo.add(Chars.toString(columnName));
```

Comment:
creating strings might not be necessary here. Tokens coming from tokenized can be made immutable until tokenizer is reset. The latter occurs after SQL compilation is fully done.

```
GenericLexer.immutableOf(tok)
```

Reply:
Marking this one as resolved as the compiler is kept around now: see b2f878f066778a7de84f262eb6779a2a568daa43